External/FEXCore/include/FEXCore/IR/IntrusiveIRList.h

```diff
@@ -20,19 +21,12 @@ namespace FEXCore::IR {
  *
  * Can potentially support reallocation if we are smart and make sure to invalidate anything holding a true pointer
  */
-class DualIntrusiveAllocator final {
+class DualIntrusiveAllocator {
   public:
     DualIntrusiveAllocator() = delete;
```

Comment:
Aren't these two constructors implicitly deleted due to providing the `size_t` constructor?

Reply:
You right. Resolved.